src/Storefront/Resources/views/storefront/element/cms-element-sidebar-filter.html.twig

```diff
@@ -28,8 +28,9 @@
             {% block element_sidebar_filter_panel %}
                 <div
                     id="filter-panel-wrapper"
-                    class="filter-panel-wrapper"
+                    class="d-none d-lg-block"
```

Comment:
If we use utility classes we should keep the selector classes in place so they can still be customized. 

In this case, we could probably keep the CSS of `filter-panel-wrapper` if we can avoid the JS of `classesToRemoveFromWrapperInOffCanvas`?